src/components/ui/atoms/Btn/Btn.jsx

```diff
@@ -16,12 +16,13 @@ export const BtnLight = ({ color, onClick, title, disabled }) => {
     <Button
       type='submit'
       fullWidth
-      variant='outlined'
+      variant={!variant ? "outlined" : variant}
```

Comment:
para que se hace esta validacion? evitemos usar las negaciones ! porque se hace un poco dificil de comprender la logica, 
mejor usemos algo como
```suggestion
      variant={variant && "outlined" }
```

Reply:
Ok, me pareme mucho mejor, lo tendre encuenta.